### PR TITLE
Upgrade SDK and Fix usage

### PR DESCRIPTION
**Pull request by cacampbell**

### CODE REVIEW

File: src/components/IDInput.vue [hunk 1]
```diff
@@ -90,31 +90,31 @@ export default createComponent({
             if (valid.value) {
                 const parts = state.input.split(".");
                 if (props.file) {
-                    state.file = {
-                        shard: parseInt(parts[0]),
-                        realm: parseInt(parts[1]),
-                        file: parseInt(parts[2])
-                    };
+                    state.file = FileId.fromString(
```
Comment (mehcode) [inline src/components/IDInput.vue]: You can remove the `parseInt` calls here

File: src/views/AccessMyAccount.vue [hunk 2]
```diff
@@ -277,20 +277,17 @@ export default createComponent({
...
 
-                await new CryptoTransferTransaction(client)
+                await new CryptoTransferTransaction()
                     .addSender(account, 0)
                     .addRecipient(recipient, 0)
-                    .setTransactionFee(1)
+                    .setMaxTransactionFee(1)
                     .build()
-                    .executeForReceipt();
+                    .execute(client);
```
Comment (mehcode) [inline src/views/AccessMyAccount.vue]: This needs to be `.getReceipt` after execute

File: src/components/IDInput.vue (hunk 1 above)
Comment (cacampbell) [reply src/components/IDInput.vue]: :+1:

**Comment by mehcode** [comment]
Manually copied in a lot of this. I probably missed any changes that you made vs. adjustments that _had_ to be made. If you still want those, please collect and re-submit.

**Pull request closed without merging**